crates/runtime/src/traphandlers/unix.rs

```diff
@@ -105,7 +105,26 @@ unsafe extern "C" fn trap_handler(
             libc::SIGSEGV | libc::SIGBUS => Some((*siginfo).si_addr() as usize),
             _ => None,
         };
-        info.set_jit_trap(pc, fp, faulting_addr);
+
+        // TODO: ideally, get these code values from libc somehow
+        // Magic numbers taken from: https://source.android.com/docs/security/test/memory-safety/mte-reports
+        // MTE synchronous mode
+        const SEGV_MTESERR: i32 = 9;
+        // MTE asynchronous mode
+        const SEGV_MTEAERR: i32 = 8;
```

Comment:
Not ideal of course. Maybe these values are in https://rust-lang.github.io/libc/aarch64-unknown-linux-gnu/doc/libc/index.html or sth similar.